.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -709,6 +724,7 @@ jobs:
       run-e2e-adder-contract-test,
 #      run-e2e-failing-version-upgrade,
 #      run-e2e-version-upgrade-catchup,
+      run-e2e-finality-version-change,
```

Comment:
Silly question – what about the two tests above? Do they still not work? Is there a task for recreating them?

Reply:
There's that https://cardinal-cryptography.atlassian.net/browse/A0-1644?search_id=5a378e7d-c138-4db0-b19a-e976662cea2d. I pinged @fixxxedpoint since he wrote the initial tests.